streamlit_folium/__init__.py

```diff
@@ -113,10 +112,8 @@ def _get_siblings(fig: folium.MacroElement) -> str:
     html = ""
     if len(children) > 1:
         for child in children[1:]:
-            try:
+            with contextlib.suppress(Exception):
```

Comment:
Are these actually equivalent?

Reply:
Yeah, just more explicit about the fact that you're really just going to ignore any exceptions in this block https://docs.astral.sh/ruff/rules/suppressible-exception/